nixos/modules/services/web-servers/nginx/vhost-options.nix

```
@@ -180,6 +180,35 @@ with lib;
       description = "Path to root SSL certificate for stapling and client certificates.";
     };
 
+    sslCiphers = mkOption {
+      type = types.nullOr types.str;
+      default = null;
+      example = "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384";
+      description = "Ciphers to choose from when negotiating TLS handshakes.";
+    };
+
+    sslProtocols = mkOption {
+      type = types.nullOr types.str;
+      default = null;
+      example = "TLSv1 TLSv1.1 TLSv1.2 TLSv1.3";
+      description = "Allowed TLS protocol versions.";
+    };
+
+    sslDhparam = mkOption {
+      type = types.nullOr types.path;
+      default = null;
+      example = "/path/to/dhparams.pem";
+      description = "Path to DH parameters file.";
+    };
+
+    recommendedTlsSettings = mkOption {
+      default = false;
+      type = types.bool;
+      description = "
+        Enable recommended TLS settings.
+      ";
```

```suggestion
      description = "Enable recommended TLS settings.";
```
Should probably not be a multi-line string.

---

Updated, fixed the original as well